src/github.com/getlantern/flashlight/client/handler.go

```diff
@@ -69,7 +73,11 @@ func (client *Client) intercept(resp http.ResponseWriter, req *http.Request) {
 		return
 	}
 
-	defer connOut.Close()
+	defer func() {
+		if err := connOut.Close(); err != nil {
+			log.Debugf("Error closing the out connection: %s", err)
+		}
+	}()
```

Comment:
Oh, by the way, I took the OK response out of the piping function to keep this last one doing just that, but that's a different topic, and I can of course put it back :)